helper/memory_calculator.go

```diff
@@ -94,7 +97,7 @@ func (m MemoryCalculator) Execute() (map[string]string, error) {
 	totalMemory := UnsetTotalMemory
 
 	if b, err := ioutil.ReadFile(m.MemoryLimitPath); err != nil && !os.IsNotExist(err) {
-		return nil, fmt.Errorf("unable to read %s\n%w", m.MemoryLimitPath, err)
+		m.Logger.Info("Unable to read %s\n%w", m.MemoryLimitPath, err)
```

Comment:
```suggestion
		m.Logger.Info("WARNING: Unable to read %s: %s", m.MemoryLimitPath, err)
```
Sorry, missed this on the first pass. `%w` only works with [Errorf](https://golang.org/src/fmt/errors.go#L17).